client/my-sites/plugins/controller.js

```diff
@@ -50,13 +50,20 @@ function renderSinglePlugin( context, siteUrl, isWpcomPlugin ) {
 	}
 
 	if (
-		( ! site || ! site.jetpack ) &&
+		( site && ! site.jetpack ) &&
```

Comment:
heh, nice, when I wrote that ( ! site || ! site.jetpack ) wasn't my most clever day :D